tests/integration/gateway_clients/test_clients_gateways.py

```diff
@@ -81,7 +81,9 @@ def complete_graph_dict():
 
 
 class DummyNoDocAccessMockConnectionPool:
-    def send_requests_once(self, requests, pod: str, head: bool) -> asyncio.Task:
+    def send_requests_once(
+        self, requests, pod: str, head: bool, endpoint: str = None
```

Comment:
```suggestion
        self, requests, pod: str, head: bool, endpoint: Optional[str] = None
```
or maybe not so important inside tests

Reply:
for the test not needed I think, specifically because its always None here actually and not uses as a string